src/Swift.Runtime/src/Swift/SwiftArray.cs

```diff
@@ -27,9 +28,11 @@ public class SwiftArray<Element> : IDisposable, ISwiftObject
 
     static nuint _elementSize = ElementTypeMetadata.Size;
 
-    private SwiftHandle _buffer;
+    private SwiftHandle _payload;
 
-    private bool _disposed = false;
+    public SwiftHandle Payload => _payload;
+
+    public IntPtr Buffer => Marshal.PtrToStructure<IntPtr>(_payload.Handle);
```

Comment:
This is entirely unnecessary, this should be a cast at best.

Reply:
Yes, removed.